docs/quickstart-docker.md

```diff
@@ -0,0 +1,141 @@
+# Start a docker container and take a backup
+
+Percona XtraBackup works in combination with a database server, Percona Server, in our scenario. To use Percona XtraBackup, you need to run it in its own Docker container and connect it to the Percona Server container. As soon as you [start Percona Server in a docker container](https://docs.percona.com/percona-server/8.0/quickstart-docker.html) and [create a database and table in Percona Server](https://docs.percona.com/percona-server/8.0/quickstart-database.html), you are ready to backup your databes with Percona XtraBackup.
```

Comment:
works in combination with a MySQL-compatible database

Reply:
fixed